oio/xcute/jobs/mover.py

```diff
@@ -21,27 +21,92 @@
+from oio.xcute.common.job import XcuteJob, XcuteTask
+
+
+class RawxDecommissionTask(XcuteTask):
+
+    def __init__(self, conf, job_config, logger=None):
+        super(RawxDecommissionTask, self).__init__(
+            conf, job_config, logger=logger)
+
+        self.service_id = job_config['service_id']
+        self.rawx_timeout = float(job_config['rawx_timeout'])
+        self.min_chunk_size = int(job_config['min_chunk_size'])
+        self.max_chunk_size = int(job_config['max_chunk_size'])
+        excluded_rawx_param = job_config['excluded_rawx']
+        if excluded_rawx_param:
+            self.excluded_rawx = excluded_rawx_param.split(',')
+        else:
+            self.excluded_rawx = list()
+
+        self.blob_client = BlobClient(
+            self.conf, logger=self.logger)
+        self.content_factory = ContentFactory(self.conf)
+        self.conscience_client = ConscienceClient(
+            self.conf, logger=self.logger)
+
+        self.fake_excluded_chunks = self._generate_fake_excluded_chunks(
+            self.excluded_rawx)
+
+    def _generate_fake_excluded_chunks(self, excluded_rawx):
+        fake_excluded_chunks = list()
+        fake_chunk_id = '0'*64
+        for service_id in excluded_rawx:
+            service_addr = self.conscience_client.resolve_service_id(
+                'rawx', service_id)
+            chunk = dict()
+            chunk['hash'] = '0000000000000000000000000000000000'
+            chunk['pos'] = '0'
+            chunk['size'] = 1
+            chunk['score'] = 1
+            chunk['url'] = 'http://{}/{}'.format(service_id, fake_chunk_id)
+            chunk['real_url'] = 'http://{}/{}'.format(service_addr,
+                                                      fake_chunk_id)
+            fake_excluded_chunks.append(chunk)
+        return fake_excluded_chunks
+
+    def process(self, chunk_id, task_payload):
+        chunk_url = 'http://{}/{}'.format(self.service_id, chunk_id)
+        meta = self.blob_client.chunk_head(chunk_url,
+                                           timeout=self.rawx_timeout)
+        container_id = meta['container_id']
+        content_id = meta['content_id']
+        chunk_size = int(meta['chunk_size'])
+
+        # Maybe skip the chunk because it doesn't match the size constaint
+        if chunk_size < self.min_chunk_size:
+            self.logger.debug("SKIP %s too small", chunk_url)
+            return {'skipped': 1}
+        if self.max_chunk_size > 0 and chunk_size > self.max_chunk_size:
+            self.logger.debug("SKIP %s too big", chunk_url)
+            return {'skipped': 1}
+
+        # Start moving the chunk
+        try:
+            content = self.content_factory.get(container_id, content_id)
+            content.move_chunk(
+                chunk_id, fake_excluded_chunks=self.fake_excluded_chunks)
+        except (ContentNotFound, OrphanChunk):
+            return {'orphan': 1}
+
+        return {'moved_chunks': 1, 'total_size': chunk_size}
```

Comment:
Lacking some consistency with the names: `skipped`, `orphan` but `moved_chunks`?